_migrations/migration-phases/backfill.md

```diff
@@ -0,0 +1,174 @@
+5. Once ready to migrate all clients, detach the Migration Assistant ALB target group from the testing NLB listener and modify the existing NLB listener to direct traffic to this target group.
+6. Now, client requests will be routed through the proxy (once they establish a new connection). Verify application metrics.
+
+### If you are using **NLB → Cluster**
+
+1. If you do not wish to modify application logic, add an ALB in front of your cluster and follow the **NLB → ALB → Cluster** steps. Otherwise:
+2. Create a target group for the ALB on port 9200 and set the health check to HTTPS.
+3. Associate this target group with your existing NLB on a new listener.
+4. Verify the health check is successful, and perform smoke testing with some clients through the new listener port.
+5. Once ready to migrate all clients, deploy a change so that clients hit the new listener.
+   
+
+### If you are **not using an NLB**
+
+Make a client/DNS change to route clients to the Migration Assistant ALB on port `9200`.
+
+
+### Kafka connection
+
+After you have routed the client based on your use case, test adding records against HTTP requests using the following steps:
+
+1. In the Migration Console, run the following command:
+
+   ```shell
+   console kafka describe-topic-records
+   ```
+   
+   Note the records in the logging topic.
+   
+2. After a short period, execute the same command again and compare the increase in records against the expected HTTP requests.
+
+
+## Creating a snapshot
+
+Create a snapshot for your backfill using the following command:
+
+```bash
+console snapshot create
+```
+
+To check the progress of your snapshot in realtime, use the following command:
+
+```bash
+console snapshot status --deep-check
+```
+
+Depending on the size of the data on the source cluster and the bandwidth allocated for snapshots, the process can take some time. Adjust the maximum rate at which the source cluster's nodes create the snapshot using the `--max-snapshot-rate-mb-per-node` option. Increasing the snapshot rate will consume more node resources, which may affect the cluster's ability to handle normal traffic.
+
+## Backfilling documents to the source cluster
+
+From the snapshot you created of your source cluster, you can begin backfilling documents to the target cluster. Once starting this process, a fleet of workers will spin up to read the snapshot and reindex documents on the target cluster. This fleet of workers can be scaled to increased the speed that documents are reindexed onto the target cluster.
+
+### Checking the starting state of the clusters
+
+You can see the indices and rough document counts of the source and target cluster by running the cat-indices command.  This can be used to monitor the difference between the source and target for any migration scenario.  Check the indices of both clusters with the following command:
```

Comment:
```suggestion
You can see the indexes and document counts of the source and target cluster by running the `cat-indices` command. This can be used to monitor the difference between the source and target for any migration scenario. Check the indexes of both clusters with the following command:
```